pkgs/applications/networking/p2p/lokinet/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, curl
+, libevent
+, libsodium
+, libuv
+, pkgconfig
+, sqlite
+, unbound
+, writeShellScriptBin
+, zeromq
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.9.9";
+  githash = "440b547d2c3e7384ec82ec21f0b1635e42bcda64";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "lokinet";
+    rev = "v${version}";
+    sha256 = "AaGsRg9S9Cng9emI/mN09QSOIRbE+x3916clWAwLnRs=";
+    fetchSubmodules = true;
+  };
+
+  fakegit = writeShellScriptBin "git" "echo ${githash}";
+
+  nativeBuildInputs = [
+    cmake
+    curl
+    fakegit
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libevent
+    libuv
+    libsodium
+    sqlite
+    unbound
+    zeromq
+  ];
+
+  cmakeFlags = [
+    "-DWITH_BOOTSTRAP=OFF" # we provide bootstrap files manually
+    "-DWITH_LTO=OFF"
```

Comment:
Does LTO not work?

Reply:
It doesn't.

Reply:
Actually, it seems to work now with 0.9.9. Not sure what changed from 0.9.8.

Reply:
Actually it seems to work now with 0.9.9.